### PR TITLE
common-instancetypes: Increase node memory in functest

### PR DESCRIPTION
**Pull request by lyarwood**
This doubles the node memory from the default of 4096 to 8192 so we can start VirtualMachines using the u1.medium instance type that itself requires 4Gi.

### CODE REVIEW

**Comment by lyarwood** [comment]
/cc @0xFelix 

Can you approve this when you get a chance?

**Review by brianmcarey — APPROVED** [review]
/approve

**Comment by brianmcarey** [comment]
/hold

Feel free to unhold

**Pull request merged**
